app/Models/Support.php

```diff
@@ -213,5 +213,76 @@ public static function promoteDelegatesToDirect($topicNum, $nickNames)
 
         return;
     }
-    
+
+    public static function getAllSupporters($topic,$camp,$excludeNickID){
```

Comment:
Not a big issue but just seems to be identical, please move the curly braces in the next line.